_posts/2018-01-18-From-Manual-to-Automatic.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+authors: Hugo Fernandes, José Ferreira, Sofia Gomes, Elaine Navalho, João Ventura
+title: From Manual to Automatic
+---
+
+In the more recent years the interest on test automation has been increased significantly. Some of the reasons behind this growth include a greater need for faster software delivery processes, increased productivity and cost/effort reduction associated to the tests itself. This article will focus on test automation, its advantages and drawbacks and when and how should the transition from manual to automatic be made. 
```

Comment:
> In the more recent years
I'd replace with "In recent years".

> and how should the transition from manual to automatic be made. 
I think the verb should come after the subject: "and how the transition from manual to automatic should be made"

Reply:
"(...) the interest on test automation has been increased significantly." it looks like it was done by someone, instead of something that happened naturally.
I would change it to:
"(...) the interest on test automation has ~~been~~ increased significantly."